### PR TITLE
V22.2.1

### PR DESCRIPTION
**Pull request by SrikanthPagadarai**

### CODE REVIEW

File: CI/scripts/build_bsp.sh [hunk 1]
```diff
@@ -104,6 +104,9 @@ cp -r scripts/boot/* ../hdl/vendor/AnalogDevices/vivado/projects/common/boot/
 
 # Copy additional IP files
 cp -r scripts/library/* ../hdl/vendor/AnalogDevices/vivado/library/
+sed -i 's@source ../../scripts/adi_env.tcl@source ../../../scripts/adi_env.tcl@' $TARGET/library/util_sync/sync_delay/util_delay_ip.tcl
+sed -i 's@source ../../scripts/adi_env.tcl@source ../../../scripts/adi_env.tcl@' $TARGET/library/util_sync/sync_fast_to_slow/sync_fast_to_slow_ip.tcl
+sed -i 's@source ../../scripts/adi_env.tcl@source ../../../scripts/adi_env.tcl@' $TARGET/library/util_sync/sync_slow_to_fast/sync_slow_to_fast_ip.tcl
```
Comment (tfcollins) [inline CI/scripts/build_bsp.sh:109]: This should be done in the source files themselves since it will make them easier to upstream.

**Review by tfcollins — CHANGES_REQUESTED** [review]
CI issues point to missing update of custom files to include new "scripts" folder in plugin_rd.m files in CustomFiles settings

File: CI/gen_doc/docs/_pages/install.md [hunk 2]
```diff
@@ -23,8 +23,8 @@ The base dependencies for the toolbox requires libiio and the libiio MATLAB bind
 
 To leverage HDL-Coder and the reference designs with the toolbox requires the following components:
 
-- Xilinx Vivado 2021.1
-- Xilinx SDK 2021.1
+- Xilinx Vivado 2021.2
+- Xilinx SDK 2021.2
```
Comment (tfcollins) [inline CI/gen_doc/docs/_pages/install.md:27]: This should reference Vitis as the SDK doesn't exist anymore

File: CI/scripts/build_bsp.sh (hunk 1 above)
Comment (SrikanthPagadarai) [reply CI/scripts/build_bsp.sh]: Fixed.

**Pull request merged**